Template/Tag/EtrackerTag.php

```diff
@@ -39,7 +44,7 @@ public function getParameters()
                 $field->description = 'Assign a etracker configuration in order to track data into a specific site.';
```

Comment:
@ben-dw We have also moved the titles and description in `en.json` so that this could be translated.
Can you merge the latest code in your branch and move the `title` and `description` hard coded into `lang/en.json` and use `Piwik::translate()`, Refer this links 

https://github.com/matomo-org/tag-manager/blob/4.x-dev/Template/Tag/EtrackerTag.php#L26

https://github.com/matomo-org/tag-manager/blob/4.x-dev/lang/en.json#L704